system/HTTP/ResponseTrait.php

```diff
@@ -398,8 +398,22 @@ public function sendHeaders()
         header(sprintf('HTTP/%s %s %s', $this->getProtocolVersion(), $this->getStatusCode(), $this->getReasonPhrase()), true, $this->getStatusCode());
 
         // Send all of our headers
-        foreach (array_keys($this->headers()) as $name) {
-            header($name . ': ' . $this->getHeaderLine($name), false, $this->getStatusCode());
+        foreach ($this->headers() as $name => $value) {
+            if ($value instanceof Header) {
+                header(
+                    $name . ': ' . $value->getValueLine(),
+                    false,
+                    $this->getStatusCode()
+                );
+            } else {
+                foreach ($value as $header) {
+                    header(
+                        $name . ': ' . $header->getValueLine(),
+                        false,
+                        $this->getStatusCode()
+                    );
+                }
+            }
```

Comment:
```suggestion
        foreach (array_keys($this->headers()) as $name)
        {
            $headerValue = $this->getHeader($name)->getValue();
            if(is_array($headerValue)) {
                foreach ($headerValue as $h_value) {
                    header($name . ': ' . $h_value, false, $this->getStatusCode());
                }
            } else {
                header($name . ': ' . $this->getHeaderLine($name), false, $this->getStatusCode());
            }
        }
```

Reply:
I don't know why my code does not work for you.
It seems your code does the same thing.

What was wrong? Any error message?